pkg/e2e/openshift/config.go

```diff
@@ -99,9 +102,19 @@ func region() string {
 }
 
 func cloud() string {
+	if viper.GetString(config.CloudProvider.CloudProviderID) == "gcp" {
+		return "gce"
+	}
 	return viper.GetString(config.CloudProvider.CloudProviderID)
 }
 
+func gcpproject() string {
+	if viper.GetString(config.CloudProvider.CloudProviderID) == "gcp" {
+		return ",\"projectid\":\"" + viper.GetString(ocmprovider.GCPProjectID) + "\""
```

Comment:
```suggestion
		return fmt.Sprintf(`,"projectid":%q`,  viper.GetString(ocmprovider.GCPProjectID) 
```

this may be a bit clearer

Reply:
that is what the `%q` directive does for us! `%s` for strings `%q` for quoted strings https://pkg.go.dev/fmt

Reply:
```
❯ go run main.go
""test""
❯ cat -p -p main.go
package main

import "fmt"

func main() {
	fmt.Printf(`"%q"`, "test")
}
```